content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -118,520 +118,6 @@ the default) is used in the service entry below.
 $ kubectl delete serviceentry wikipedia
 {{< /text >}}
 
-## Configure egress gateway traffic to a wildcard host
-
-The configuration for accessing a wildcard host via an egress gateway depends on whether or not
-the set of wildcard domains are served by a single common host.
-This is the case for _*.wikipedia.org_. All of the language-specific sites are served by every
-one of the _wikipedia.org_ servers. You can route the traffic to an IP of any _*.wikipedia.org_ site,
-including _www.wikipedia.org_, and it will [manage to serve](https://en.wikipedia.org/wiki/Virtual_hosting)
-any specific site.
-
-In the general case, where all the domain names of a wildcard are not served by a single hosting server,
-a more complex configuration is required.
-
-### Wildcard configuration for a single hosting server
```

Comment:
Does this section need to be deleted too? Seems that it's really only the next section that uses the nginx proxy and is the overly complex config that we don't want to recommend.

Reply:
Good call, added it back